com.microsoft.mrtk.input/Interactors/InteractorVisuals/Lines/Definitions/InterpolationType.cs

```diff
@@ -8,8 +8,14 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public enum InterpolationType
     {
+        /// <summary>
+        /// Interpolation using Bézier parametric curve.
+        /// </summary>
         Bezier = 0,
-        CatmullRom,
-        Hermite,
```

Comment:
Was Hermite unused?

Reply:
I have never heard any of these words before...

Reply:
Yup :) Unused... see PR description.

Reply:
Oop sorry